package.json

```diff
@@ -4,18 +4,42 @@
   "description": "Back-End Repo for Team 1 of Baboon/Bald Eagle Practicum",
   "main": "app.js",
   "scripts": {
-    "dev": "nodemon src/server.js"
+    "dev": "nodemon src/app.js",
+    "start": "node app.js",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "format": "prettier --write ."
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "argon2": "^0.30.3",
+    "bcryptjs": "^2.4.3",
+    "body-parser": "^1.20.2",
+    "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
+    "dotenv": "^16.3.1",
     "express": "^4.18.2",
+    "express-async-errors": "^3.1.1",
     "express-favicon": "^2.0.4",
-    "mongoose": "^7.0.1",
-    "morgan": "^1.10.0"
+    "express-rate-limit": "^6.8.1",
+    "express-session": "^1.17.3",
+    "express-xss-sanitizer": "^1.1.6",
+    "helmet": "^7.0.0",
+    "jsonwebtoken": "^9.0.1",
+    "mongo-sanitize": "^1.1.0",
+    "mongoose": "^7.4.1",
+    "morgan": "^1.10.0",
+    "passport": "^0.6.0",
+    "validator": "^13.9.0",
+    "winston": "^3.10.0",
+    "xss-clean": "^0.1.4"
   },
   "devDependencies": {
-    "nodemon": "^2.0.21"
+    "eslint": "^8.46.0",
+    "eslint-config-prettier": "^8.9.0",
+    "eslint-plugin-prettier": "^5.0.0",
```

Comment:
I don't believe you need `eslint-plugin-prettier`. Just `eslint-config-prettier` should be enough to have ESLint and Prettier work well together

Reply:
eslint-plugin-prettier removed , commit bb4a51e79cbcd88ffdaa4e080689113269222f95